### PR TITLE
Provide BookieId option for listledgers command

### PR DESCRIPTION
**Pull request by reddycharan**
Descriptions of the changes in this PR:

### Motivation

While diagnosing/analyzing/debugging cluster, sometimes it would be helpful to know the list of the ledgers residing in a particular bookie. Currently there is no simple way (BookieShell command) to find out that.

### Changes

For listledgers BookieShell command, provide bookieid option, which helps us in knowing list of ledgers residing in this particular Bookie.

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java [hunk 1]
```diff
@@ -970,48 +973,87 @@ int runCmd(CommandLine cmdLine) throws Exception {
...
+                                if (!printMeta && (bookieAddress == null)) {
+                                    printLedgerMetadata(ledgerId, null, false);
+                                    cb.processResult(BKException.Code.OK, null, null);
+                                } else {
+                                    GenericCallback<LedgerMetadata> gencb = new GenericCallback<LedgerMetadata>() {
+                                        @Override
+                                        public void operationComplete(int rc, LedgerMetadata ledgerMetadata) {
+                                            if (rc == BKException.Code.OK) {
+                                                if ((bookieAddress == null)
+                                                        || bkAdmin.areEntriesOfLedgerStoredInTheBookie(ledgerId,
```
Comment (sijie) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java]: I think `areEntriesOfLedgerStoredInTheBookie` and `areEntriesOfSegmentStoredInTheBookie` can be static. If you make them static, we don't need to construct another `BookKeeperAdmin` object.

**Comment by eolivelli** [comment]
retest this please

**Comment by sijie** [comment]
retest this please

**Comment by eolivelli** [comment]
rerun bookkeeper-server bookie tests
rerun integration tests

**Comment by sijie** [comment]
rerun bookkeeper-server bookie tests

**Comment by reddycharan** [comment]
@sijie done.

**Comment by sijie** [comment]
rerun bookkeeper-server client tests
rebuild java8

**Comment by sijie** [comment]
rerun bookkeeper-server client tests

**Pull request closed without merging**